LEAF_Request_Portal/js/vue_conditions_editor/LEAF_conditions_editor.js

```diff
@@ -541,23 +451,100 @@ ConditionsEditor.component('editor-main', {
         isOrphan(childIndID) {
             return !this.selectableParents.some(p => parseInt(p.indicatorID) === parseInt(childIndID));
         },
-        childFormatChangedSinceSave(condition){
+        childFormatChangedSinceSave(condition) {
             const childConditionFormat = condition.childFormat;
-            const childID = parseInt(condition.childIndID);
-            const childIndicator = this.indicators.find(ind => parseInt(ind.indicatorID) === childID);
-            const currentIndicatorFormat = childIndicator?.format?.split('\n')[0];
+            const currentIndicatorFormat = this.childIndicator?.format?.split('\n')[0];
             return childConditionFormat?.trim() !== currentIndicatorFormat?.trim();
+        },
+        updateChoicesJS() {
+            const elExistingChoicesChild = document.querySelector('#outcome-editor > div.choices');
+            const elSelectParent = document.getElementById('parent_compValue_entry');
+            const elSelectChild = document.getElementById('child_prefill_entry');
+            
+            const childFormat = this.conditions.childFormat.toLowerCase();
+            const parentFormat = this.conditions.parentFormat.toLowerCase();
+            const outcome = this.conditions.selectedOutcome.toLowerCase();
+           
+            if(parentFormat === 'multiselect' && elSelectParent !== null && !elSelectParent.choicesjs) {

```

Comment:
Just a note for future consideration:  We're going to want to have a map for these selections so we can compare ints instead of strings.